params/config.go

```diff
@@ -71,6 +71,7 @@ var (
 			BlockPeriod:    5,
 			RequestTimeout: 3000,
 			LookbackWindow: 12,
+			V2Block:        big.NewInt(15287159),
```

Comment:
These activation blocks will need to be set again, the mainnet is only 4 days from now but I see that baklava has already happened. (didn't check alfajores)

Reply:
These were placeholder values I set a long time ago, yes, we need to think what the proper date will be